src/hints/mock_call/mod.rs

```diff
@@ -16,35 +16,35 @@ use num_bigint::BigInt;
 
 pub const MOCK_CALL_KEY: &str = "mock_call";
 
-/// Mocks the return value of a specific function.
+/// `mock_call` is designed to be a hint, executable by the HintExecutor of the cairoVM
```

Comment:
Keep what was already the title, it should be bref. And bellow, add this sentence

Reply:
> https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html Here is a good ressource. Must read if you want to do more documentation work

Yes I must go through this